packages/components/src/date-time/index.js

```diff
@@ -1,24 +1,30 @@
 /**
  * Internal dependencies
  */
+import 'react-dates/initialize';
+import 'react-dates/lib/css/_datepicker.css';
```

Comment:
CSS in here? It will break package published to npm.

Reply:
Huh, right, I didn't even think of that. I'm still adjusting to lerna.

Reply:
We changed the process as before we were importing CSS with Webpack and didn't care too much about npm ecosystem. It still is supported as you could find out :) 
It's less Lerna thing but more the necessity to use only JS code so you could consume it from npm without Webpack and similar tools.